### PR TITLE
Guard against a NP exception and updated over provisioning flags in README.

### PR DESCRIPTION
**Pull request by maselvaraj**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -136,6 +136,12 @@ At a minimum, a container path must be entered to mount the volume. A host path
 
 Additional parameters are available for the `docker run` command, but there are too many and they change too often to list all separately. This section allows you to provide any parameter you want. Ensure that your Docker version on your Mesos slaves is compatible with the parameters you add and that the values are correctly formatted. Use the full-word parameter and not the shortcut version, as these may not work properly. Also, exclude the preceding double-dash on the parameter name. For example, enter `volumes-from` and `my_container_name` to recieve the volumes from `my_container_name`. Of course `my_container_name` must already be on the Mesos slave where the Jenkins slave will run. This shouldn't cause problems in a homogenous environment where Jenkins slaves only run on particular Mesos slaves.
 
+### Over provsioning flags ###
```
Comment (geoffroyjabouley) [inline README.md]: provsioning --> provisioning?

**Comment by geoffroyjabouley** [comment]
Looks good to me

File: README.md (hunk 1 above)
Comment (maselvaraj) [inline README.md]: Thanks ! I've corrected it.

**Comment by bacoboy** [comment]
Now that most of the outstanding PR's have been merged.  Can we get an official 0.8.0 release?

Might also want to update the "Latest Release" badge on the https://github.com/jenkinsci/mesos-plugin/releases page.

Thanks!

**Comment by vinodkone** [comment]
Yea. We plan to cut a 0.8.0 release next week.

**Pull request merged**